### PR TITLE
Order confirm mutation

### PR DESCRIPTION
**Pull request by tomaszszymanski129**
I want to merge this change because it adds a mutation to confirm an order.

_This is a PR to feature branch._

Related ticket: [SALEOR-1407](https://app.clickup.com/t/2549495/SALEOR-1407)

# Impact

* [X] New migrations
* [X] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [X] The changes are tested
* [X] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1407 Add mutation to confirm an order](https://app.clickup.com/t/2549495/SALEOR-1407)

File: saleor/graphql/order/mutations/orders.py [hunk 1]
```diff
@@ -483,3 +483,39 @@ def perform_mutation(cls, _root, info, amount, **data):
...
+                        code=OrderErrorCode.INVALID,
+                    )
+                }
+            )
+        return instance
+
+    @classmethod
+    def perform_mutation(cls, root, info, **data):
+        order = cls.get_instance(info, **data)
+        order.status = OrderStatus.UNCONFIRMED
```
Comment (IKarbowiak) [inline saleor/graphql/order/mutations/orders.py:519]: As I see from mutation's description it should be:
``` suggestion
        order.status = OrderStatus.UNFULFILLED
```

File: saleor/graphql/order/tests/test_order.py [hunk 2]
```diff
@@ -430,6 +430,50 @@ def test_order_query_shows_confirmed_orders(
...
+    staff_api_client.user.user_permissions.add(permission_manage_orders)
+    response = staff_api_client.post_graphql(
+        ORDER_CONFIRM_MUTATION,
+        {"id": graphene.Node.to_global_id("Order", order_unconfirmed.id)},
+    )
+    order_data = get_graphql_content(response)["data"]["orderConfirm"]["order"]
+
+    assert order_data["status"] == OrderStatus.UNCONFIRMED.upper()
+    order_unconfirmed.refresh_from_db()
+    assert order_unconfirmed.status == OrderStatus.UNCONFIRMED
```
Comment (IKarbowiak) [inline saleor/graphql/order/tests/test_order.py:458]: The test also should be changed.

File: saleor/graphql/order/mutations/orders.py [hunk 3]
```diff
@@ -483,3 +483,39 @@ def perform_mutation(cls, _root, info, amount, **data):
...
+        order.save(update_fields=["status"])
+        return OrderConfirm(order=order)
```
Comment (fowczarek) [inline saleor/graphql/order/mutations/orders.py:521]: 1. Should we save any event in this mutation for example `OrderConfirmed`?
2. Should we trigger any webhook here?

Comment (tomaszszymanski129) [reply saleor/graphql/order/mutations/orders.py:521]: Yep, we should trigger a webhook, this is already done in separate PR.
Adding an event is a good idea, will do that in a moment.

**Review by IKarbowiak — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**